### PR TITLE
HLM-4367:UI/ UX Audit fixes

### PR DESCRIPTION
**Pull request by anilsingha-eGov**
Implemented UI/UX Audit Fixes in the following points [UI/UX Audit Fixes](https://docs.google.com/spreadsheets/d/10diOgTplRs5XDvjoFQVf37E3rk9WUobXkd2DBybfYpU/edit#gid=0)

### CODE REVIEW

File: apps/health_campaign_field_worker_app/lib/pages/beneficiary/beneficiary_details.dart [hunk 1]
```diff
@@ -134,9 +135,13 @@ class _BeneficiaryDetailsPageState
                           return cycles != null && cycles.isNotEmpty
                               ? deliverState.hasCycleArrived
                                   ? SizedBox(
-                                      height: 100,
+                                      height: 90,
                                       child: DigitCard(
-                                        margin: const EdgeInsets.all(kPadding),
+                                        margin: const EdgeInsets.only(
+                                          left: 0,
```
Comment (naveen-egov) [inline apps/health_campaign_field_worker_app/lib/pages/beneficiary/beneficiary_details.dart:141]: we don't need to specify  0

File: apps/health_campaign_field_worker_app/lib/pages/beneficiary/widgets/past_delivery.dart [hunk 2]
```diff
@@ -38,15 +38,33 @@ Widget buildTableContent(
...
+  final projectState = context.read<ProjectBloc>().state;
+  final item = projectState
+      .projectType!.cycles![currentCycle - 1].deliveries![currentDose - 1];
+  final productVariants =
+      fetchProductVariant(item, individualModel)?.productVariants;
+  final numRows = productVariants?.length ?? 0;
+  const rowHeight = 82;
+  const paddingHeight = kPadding / 2;
+  final containerHeight = (numRows + 1) * rowHeight + paddingHeight;
+
```
Comment (naveen-egov) [inline apps/health_campaign_field_worker_app/lib/pages/beneficiary/widgets/past_delivery.dart:52]: remove commented code

File: apps/health_campaign_field_worker_app/lib/widgets/no_result_card/no_result_card.dart [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  @override
+  Widget build(BuildContext context) {
+    return Align(
+      alignment: align,
+      child: Padding(
+        padding: const EdgeInsets.only(top: 10, bottom: 10, right: 8),
+        child: Column(
+          children: [
+            SvgPicture.asset('assets/icons/svg/no_result.svg'),
+            const SizedBox(height: 20),
```
Comment (naveen-egov) [inline apps/health_campaign_field_worker_app/lib/widgets/no_result_card/no_result_card.dart:22]: move this to constants and read from constants

**Review by naveen-egov — CHANGES_REQUESTED** [review]

File: apps/health_campaign_field_worker_app/lib/pages/beneficiary/beneficiary_details.dart (hunk 1 above)
Comment (anilsingha-eGov) [reply apps/health_campaign_field_worker_app/lib/pages/beneficiary/beneficiary_details.dart]: I have removed the 0 @naveen-egov

File: apps/health_campaign_field_worker_app/lib/pages/beneficiary/widgets/past_delivery.dart (hunk 2 above)
Comment (anilsingha-eGov) [reply apps/health_campaign_field_worker_app/lib/pages/beneficiary/widgets/past_delivery.dart:52]: Removed @naveen-egov

File: apps/health_campaign_field_worker_app/lib/pages/home.dart [hunk 4]
```diff
@@ -334,16 +334,21 @@ class _HomePageState extends LocalizedState<HomePage> {
...
       ),
-      i18.home.warehouseManagerCheckList:
-          homeShowcaseData.wareHouseManagerChecklist.buildWith(
-        child: HomeItemCard(
-          icon: Icons.checklist,
-          label: i18.home.warehouseManagerCheckList,
-          onPressed: () => context.router.push(ChecklistWrapperRoute()),
-        ),
-      ),
+      //[todo] commenting for now since both the routes are same
```
Comment (naveen-egov) [inline apps/health_campaign_field_worker_app/lib/pages/home.dart:337]: why is this code commented

File: apps/health_campaign_field_worker_app/lib/pages/home.dart [hunk 5]
```diff
@@ -419,8 +424,8 @@ class _HomePageState extends LocalizedState<HomePage> {
           homeShowcaseData.warehouseManagerManageStock.showcaseKey,
       i18.home.stockReconciliationLabel:
           homeShowcaseData.wareHouseManagerStockReconciliation.showcaseKey,
-      i18.home.warehouseManagerCheckList:
-          homeShowcaseData.wareHouseManagerChecklist.showcaseKey,
+      // i18.home.warehouseManagerCheckList:
```
Comment (naveen-egov) [inline apps/health_campaign_field_worker_app/lib/pages/home.dart:427]: pls remove the commented code

**Review by naveen-egov — CHANGES_REQUESTED** [review]

**Pull request closed without merging**